provision-contest/disable-turboboost_ht

```diff
@@ -5,7 +5,7 @@ shopt -s extglob
 
 declare -A core_ids
 
-for cpu in /sys/devices/system/cpu/cpu* ; do
+for cpu in $(ls -1d /sys/devices/system/cpu/cpu* | sort --version-sort) ; do
```

Comment:
The following command should give the same output (verified locally) and should make shellcheck happy:
`find /sys/devices/system/cpu -maxdepth 1 -mindepth 1 -name 'cpu*' | sort --version-sort`. 

The max- and mindepth flags are needed to ensure it only lists the cpu's and skips the directory itself.

Reply:
What is shellcheck unhappy about? I'm fine with suppressing an issue that is unlikely to cause harm.

Reply:
`./provision-contest/disable-turboboost_ht:8:14: note: Use find instead of ls to better handle non-alphanumeric filenames. [SC2012]`

use `find` instead of `ls`

Reply:
Pfff, that's just random advice that doesn't apply here inside `/sys/...`. Let's suppress.

Reply:
@eldering so this can be merged as is?